lib/libutils/isoc/include/stdlib.h

```diff
@@ -15,6 +15,8 @@
 #include <stddef.h>
 #include <malloc.h>
 
+/* aligend_alloc() is declared in included malloc.h */
```

Comment:
I would prefer to keep things as close to the standard as possible, that is: declare `aligned_alloc()` in `<stdlib.h>` not in `<malloc.h>` and use the proper C11 guard:
```C
#if __STDC_VERSION__ >= 201112L
void *aligned_alloc(size_t alignment, size_t size);
#endif
```

[Upstream newlib](https://github.com/bminor/newlib/blob/master/newlib/libc/include/stdlib.h#L331) uses `__ISO_C_VISIBLE` which seems to be the BSD libc way of doing things, I think it makes more sense to use `__STDC_VERSION__ defined by the compiler.